package.json

```diff
@@ -53,6 +54,7 @@
     "babel-plugin-add-header-comment": "^1.0.3",
     "babel-plugin-const-enum": "^1.2.0",
     "chokidar": "^3.5.3",
+    "compression": "^1.7.4",
```

Comment:
It seems like this is unused as you've used CompressionStream?
However if this provides stable compression results then it makes sense to use it instead.

Reply:
I think this was needed for TS to know the types, but maybe there's a better package for that. I'll double check tomorrow.

Reply:
`compression` appears to be a compression library unrelated to CompressionStream.
The CompressionStream types should be built into TypeScript (I'm seeing at least the `CompressionStream` symbol is defined).

Reply:
You're right. Removed.